packages/builder/src/internal/constants.ts

```diff
@@ -0,0 +1,14 @@
+export const BUIDLER_NAME = "Buidler";
+
+export const BUIDLER_EXECUTABLE_NAME = "buidler";
+export const BUIDLEREVM_NETWORK_NAME = "buidlerevm";
+
+export const SOLC_INPUT_FILENAME = "solc-input.json";
+export const SOLC_OUTPUT_FILENAME = "solc-output.json";
+
+export const BUIDLEREVM_SUPPORTED_HARDFORKS = [
+  "byzantium",
+  "constantinople",
+  "petersburg",
+  "istanbul",
+];
```

Comment:
everything in this file should be removed or changed

Reply:
Removed.